src/commcare_cloud/ansible/ebsnvme/tasks/main.yml

```diff
@@ -0,0 +1,25 @@
+- name: Install nvme-cli
+  become: yes
+  apt: name=nvme-cli state=present
+  notify: "Reload udev rules"
+
+- name: Download ebsnvme
+  become: yes
+  get_url:
+    url: "https://gist.githubusercontent.com/lbernail/d851e5b06eb32180a4b8ead2ce4f45db/raw/07772083fa1c2e567ed422eb401ac7783487d1c7/ebsnvme-id"
```

Comment:
Would it be prudent to copy this somewhere we have control? Like into this repo? Linking to a gist seems a bit scary to me.